docs/README.md

```diff
@@ -5,13 +5,9 @@ All documentation is in plain text files using
 [reStructuredText](http://docutils.sourceforge.net/rst.html) and
 [Sphinx](http://sphinx-doc.org/).
```

Comment:
This isn't correct any longer

Reply:
It is true, we just now also have myST as a parser that just converts the markdown to restructured text before rendering to html. Updated to add myst explicitly.

Reply:
Sure, but I read it as saying "all documentation is written using rst, rendered with these companion packages" when it's actually "all documentation, apart from API docs, are written using _markdown_, rendered with these companion packages"

Reply:
Does the updated version clarify? If not could you suggest something that is more clear?